src/CashuWallet.ts

```diff
@@ -750,34 +791,67 @@ class CashuWallet {
 		outputAmounts?: OutputAmounts,
 		counter?: number,
 		pubkey?: string,
-		privkey?: string
-	): {
-		payload: SwapPayload;
-		blindingData: BlindingData;
-	} {
+		privkey?: string,
+		customOutputData?: {
+			keep?: Array<OutputDataLike> | OutputDataFactory;
+			send?: Array<OutputDataLike> | OutputDataFactory;
```

Comment:
should there be a type for `Array<OutputDataLike> | OutputDataFactory;`? Seems to be repeated a lot, but maybe its more clear to write it out each time

Reply:
I decided to keep a union here, because IDEs will then show the union instead of the alias type. I think this is beneficial, because these two types are very different things that change the way the function behaves.